projects/plugins/jetpack/extensions/plugins/payments/payments.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Payments plugin.
+ *
+ * @since $$next-version$$
+ *
+ * @package automattic/jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Payments;
+
+use Jetpack_Gutenberg;
+
+const FEATURE_NAME = 'payments';
+
+// Populate the available extensions with our feature.
+add_filter(
+	'jetpack_set_available_extensions',
+	function ( $extensions ) {
+		return array_merge(
+			$extensions,
+			array(
+				FEATURE_NAME,
+			)
+		);
+	}
+);
+
+// Set the feature availability, depending on the site plan.
+add_action(
+	'jetpack_register_gutenberg_extensions',
+	function () {
+		Jetpack_Gutenberg::set_availability_for_plan( FEATURE_NAME );
```

Comment:
@jeherve Could you please confirm if `set_availability_for_plan()` for feature name `payments` will enable this plugin for sites hosted on WP.com? Checking because [this function](https://github.com/Automattic/jetpack/blob/a27793f036312183795838b0ee8c0c03da7a9cef/projects/plugins/jetpack/class.jetpack-gutenberg.php#L1118) does not seem to map to a feature called `payments`, and only has simple-payments and recurring-payments but maybe I'm missing something.

Reply:
You will indeed need to update WordPress.com as well. This is why I had added this to the PR description:

> On WordPress.com, this will require changes to add that new Payments feature to the list mapping plans and features.

Reply:
Ah sorry, I missed that. Thanks for clarifying.